src/movie-component.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./src/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>MOVIE COMPONENT</title>
+</head>
+<body>
+    <comp-movie></comp-movie>
```

Comment:
Jezeli dodam wiecej niz jeden tag to nadal sie wyswietala tylko jeden na ekranie. W html za to widnieja wszystkie ale puste w srodku poza jednym